### DISCUSSION TITLE
Word Spacing Utilities

### DISCUSSION THREAD

**Discussion by slavarazum** [tailwindlabs/tailwindcss#2874] [Ideas]
It would be great to have utilities for `word-spacing` css property by analogy with letter spacing utilities.
I have searched through typography terms but haven't found any to define word spacing as **tracking** for letter spacing. Maybe someone will have an idea?

**Comment by cilex-ft** [comment]
spacer?

**Comment by slavarazum** [comment]
@adamwathan 
Maybe **interval**.
What do you think, may I create a PR?

**Comment by tesar-tech** [comment]
We can do `class="[word-spacing:0.16rem]"` with arbitrary values, but I am bit surprised it still doesn't exists as an utility..
